test/integration/test_uv_append.c

```diff
@@ -65,19 +67,23 @@ static void appendCbAssertResult(struct raft_io_append *req, int status)
         }                                                   \
     }
 
-/* Submit an append request identified by I, with N_ENTRIES entries, each one of
- * size ENTRY_SIZE. The default expectation is for the operation to succeed. A
- * custom STATUS can be set with APPEND_EXPECT. */
-#define APPEND_SUBMIT(I, N_ENTRIES, ENTRY_SIZE)                     \
+#define APPEND_SUBMIT_CB_DATA(I, N_ENTRIES, ENTRY_SIZE, CB, DATA)   \
```

Comment:
Shouldn't this one have a description comment?

Reply:
Ok